AsyncDisplayKit/ASTextNode.mm

```diff
@@ -406,10 +408,22 @@ - (NSObject *)drawParametersForAsyncLayer:(_ASDisplayLayer *)layer
   // Offset the text origin by any shadow padding
   UIEdgeInsets shadowPadding = [self shadowPadding];
   CGPoint textOrigin = CGPointMake(self.bounds.origin.x - shadowPadding.left, self.bounds.origin.y - shadowPadding.top);
-  return [[ASTextNodeDrawParameters alloc] initWithRenderer:[self _renderer]
-                                                       shadower:[self _shadower]
-                                                     textOrigin:textOrigin
-                                                backgroundColor:self.backgroundColor.CGColor];
+
+  switch (_verticalAlignment)
+  {
```

Comment:
Code style has the opening brace on the same line as the control statement.  Although the switch would make sense if we had more than one option for this, could you fall back to a simple if (_verticalAlignment == ASTextNodeAlignmentCenter) check?

Check the return line below this — it is indented too far.  Thanks for fixing the alignment of the method arguments, as they were misaligned before!